packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts

```diff
@@ -338,6 +343,12 @@ export const actionCreators: ActionCreators = {
 
         workspace = await getDevWorkspaceClient().manageDebugMode(workspace, debugWorkspace);
 
+        const editorName = getEditorName(workspace);
+        const lifeTime = getLifeTime(workspace);
+        if (editorName && lifeTime > 30) {
```

Comment:
I'm not sure I know anything about this devworkspace life time limit. Can you please elaborate on why you chose 30 ms?

Reply:
The ```lifeTime``` units were seconds.
 I chose 30 seconds as the time when the editor is considered relevant.